### DISCUSSION TITLE
Cannot update Payment Method for Github because Edit button is not available

### DISCUSSION THREAD

**Discussion by vincent-prz** [community/community#125590] [Enterprise]
### Body

the edit button is nowhere to be seen, How do I update the payment method ? (see screenshot)

![image](https://github.com/community/community/assets/6080805/bc837cb0-46af-4014-bd91-73ad02e84680)

**Comment by queenofcorgis** [comment]
Hi @vincent-prz, 

Have you tried this on a different browser?

**Reply by vincent-prz** [reply]
Hi @queenofcorgis 

same thing on both Firefox and Chromium.. this is annoying as I need to update the payment method soon, because my card will expire. How to proceed in this case ?
